src/content/guides/whats-new.md

```diff
@@ -17,6 +17,71 @@ and follow the [Dart blog][].
 [dart-announce]: https://groups.google.com/a/dartlang.org/d/forum/announce
 [Dart blog]: https://medium.com/dartlang
 
+## February 15, 2024: 3.3 release
+
+This section lists notable changes made from November 16, 2023,
+through February 15, 2024.
+For details about the 3.3 release,
+check out [3.3 blog post][] and the [SDK changelog][3-3-changelog].
+
+[3.3 blog post]: https://medium.com/dartlang/dart-3-3-325bf2bf6c13
+[3-3-changelog]: https://github.com/dart-lang/sdk/blob/main/CHANGELOG.md#330
+
+### Docs updated or added to dart.dev {:.no_toc}
+
+In addition to bug fixes and incremental improvements,
+we made the following changes to this site:
+
+* Changed the site infrastructure to [run on 11ty][] instead of Jekyll.
+* Added a page covering the new language feature [extension types][].
+* Added a new documentation set on [JavaScript interop][]:
+  * [Usage][]
+  * [JS types][]
+  * [Tutorials][], the first of which is on [Mocks][]
+  * [Past JS interop][]
+* Updated [Concurrency][] overview, plus a new practical page on using [Isolates][]
+* Added a section about [`external`][] to the [Functions][] page.
+* Removed Dartpad from many pages to accomodate the new version of [Dartpad][].
+* Included a definition for "Function" in the [Glossary][].
+* Deconstructed the [Library tour][] into individual pages for each library.
+* Updated the [Breaking changes][] page for 3.3
+* Removed outdated FAQ page.
+* Expanded documentation on [`dart doc`][].
+* Updated and simplified [supported platforms][] content.
+* Consolidated multiples of [`dart format`][] content.
+* Moved away from recommending [`dart:html`][] and [`dart:io`][].
```

Comment:
We still recommend `dart:io`, just not for making HTTP requests. Maybe you can separate these poomts

```suggestion
* Updates various locations to suggest `package:web` instead of [`dart:html`][].
* Moved away from recommending `dart:html` and `dart:io` for making HTTP requests,
  in favor of `package:http`.
```

Reply:
I'm going to edit this one myself to add links to the package web page and differentiate the anchors for dart:html vs the specific http request section. Thanks for catching that!